sp/src/game/server/props.cpp

```diff
@@ -2034,8 +2079,9 @@ void CBreakableProp::Break( CBaseEntity *pBreaker, const CTakeDamageInfo &info )
 	{
 		if ( bExploded == false )
 		{
-			ExplosionCreate( origin, angles, pAttacker, 1, m_explodeRadius, 
-				SF_ENVEXPLOSION_NOSPARKS | SF_ENVEXPLOSION_NODLIGHTS | SF_ENVEXPLOSION_NOSMOKE, 0.0f, this );			
+			ExplosionCreate( WorldSpaceCenter(), angles, pAttacker, m_explodeDamage, m_explodeRadius,
```

Comment:
Why is the default (non-zap) explosion changed? I can see that instead of the origin local variable, it uses the world space center, and instead of 1 (?) it uses the explosion damage. It also applies the flag for surface only. Are these changes related to the zap barrels?

Reply:
No, this was a mistake. Thank you for catching that